data-esnext.js

```diff
@@ -3382,6 +3382,90 @@ exports.tests = [
   res : {
   }
 },
+{
+  name: 'BigInt',
+  category: STAGE3,
+  significance: 'medium',
+  spec: 'https://github.com/tc39/proposal-bigint',
+  subtests: [
+    {
+      name: 'basic functionality',
+      exec: function () {/*
+        return (1n + 2n) === 3n;
+      */},
+      res: {
+      },
+    },
+    {
+      name: 'constructor',
+      exec: function () {/*
+        return BigInt("3") === 3n;
+      */},
+      res: {
+      },
+    },
+    {
+      name: 'BigInt.asUintN',
+      exec: function () {/*
+        return typeof BigInt.asUintN === 'function';
+      */},
+      res: {
+      },
+    },
+    {
+      name: 'BigInt.asIntN',
+      exec: function () {/*
+        return typeof BigInt.asIntN === 'function';
+      */},
+      res: {
+      },
+    },
+    {
+      name: 'BigInt.parseInt',
+      exec: function () {/*
+        return typeof BigInt.parseInt === 'function';
+      */},
+      res: {
+      },
+    },
+    {
+      name: 'BigInt64Array',
+      exec: function () {/*
+        var buffer = new ArrayBuffer(64);
+        var view = new BigInt64Array(buffer);         view[0] = 0x8000000000000000;
+        return view[0] === -9223372036854776000n;
+      */},
+      res: {
+      },
+    },
+    {
+      name: 'BigUint64Array',
+      exec: function () {/*
+        var buffer = new ArrayBuffer(64);
+        var view = new BigUint64Array(buffer);        view[0] = 0x10000000000000000;
```

Comment:
You probably want to put an `n` on the end of that literal, unless you want it to throw a TypeError (with the current spec); ditto above.

Reply:
Thanks fixed